properties/basic.py

```diff
@@ -760,7 +762,7 @@ def unicode(self, value):
 
     @property
     def regex(self):
-        """Regular expression the string must comply with"""
+        """Regular expression the string must include"""
```

Comment:
Old language is fine with me. "comply with" or maybe "follows" (especially in your example below)? I'd prefer to avoid "matches" if we are using `search` instead of `match`.

Regarding performance, could be worth looking into sometime. In my opinion, the more generic nature of `search` overrides slight performance differences, at least given the current, not-too-intensive use-cases.